framework/error_handlers/error_4xx.py

```diff
@@ -0,0 +1,46 @@
+# External Libraries
+from flask import Response
+from werkzeug.exceptions import HTTPException
+
+# Sayonika Internals
+from framework.error_handlers.error_common import error_handler
+
+
+@error_handler
+def handle_400(err: HTTPException):
+    return Response(err.description, 400)
+
+
+@error_handler
+def handle_401(err: HTTPException):
+    return Response(err.description, 401)
+
+
+@error_handler
+def handle_403(err: HTTPException):
+    return Response(err.description, 403)
+
+
+@error_handler
+def handle_404(err: HTTPException):
+    return Response(err.description, 404)
+
+
+@error_handler
+def handle_405(err: HTTPException):
+    return Response(err.description, 405)
+
+
+@error_handler
+def handle_429(err: HTTPException):
+    return Response(f"Ratelimit for this endpoint: {err.description}", 429)
+
```

Comment:
add a 418 for the memes (ask @Ovyerus about this).

Reply:
Done